src/platform_impl/windows/window.rs

```diff
@@ -610,26 +621,29 @@ impl Window {
             return;
         }
 
-        self.thread_executor.execute_in_thread(move || unsafe {
-            let (flags, count) = request_type
-                .map(|ty| match ty {
-                    UserAttentionType::Critical => {
-                        (winuser::FLASHW_ALL | winuser::FLASHW_TIMERNOFG, u32::MAX)
-                    }
-                    UserAttentionType::Informational => {
-                        (winuser::FLASHW_TRAY | winuser::FLASHW_TIMERNOFG, 0)
-                    }
-                })
-                .unwrap_or((winuser::FLASHW_STOP, 0));
-
-            let mut flash_info = winuser::FLASHWINFO {
-                cbSize: mem::size_of::<winuser::FLASHWINFO>() as UINT,
-                hwnd: window.0,
-                dwFlags: flags,
-                uCount: count,
-                dwTimeout: 0,
-            };
-            winuser::FlashWindowEx(&mut flash_info);
+        self.thread_executor.execute_in_thread(move || {
+            let _ = &window;
+            unsafe {
```

Comment:
To have a smaller diff here, please move the `unsafe` back on the line it was before, if that makes sense

Reply:
As in `self.thread_executor.execute_in_thread(move || { let _ = &window; unsafe {` ?
I dont think rust fmt would allow that.

Reply:
I was thinking:
```rust
self.thread_executor.execute_in_thread(move || unsafe {
    let _ = &window;
```

Reply:
ah! I didnt think of that, sounds good to me